fw/Core/Hitcon/Logic/XBoardLogic.cc

```diff
@@ -128,6 +142,8 @@ void XBoardLogic::QueueDataForTx(uint8_t *packet, uint8_t packet_len,
   for (uint8_t i = 0; i < packet_len; ++i) {
     pkt[i + HEADER_SZ] = packet[i];
   }
+  reinterpret_cast<Frame *>(pkt)->checksum =
+      fast_crc32(pkt, HEADER_SZ + packet_len);
```

Comment:
fast_crc32() only handles data size that is multiple of 4 bytes. Do you want to pad all packets up to multiples of 4 or you want to use the normal crc32() function?